maps_dashboards/public/components/layer_config/layer_config_panel.tsx

```diff
@@ -19,13 +19,14 @@ import {
   EuiModalBody,
   EuiModalFooter,
   EuiModalHeader,
-  EuiModalHeaderTitle,
+  EuiModalHeaderTitle, EuiIcon
 } from '@elastic/eui';
 
 import { MapLayerSpecification } from '../../model/mapLayerType';
 import { BaseMapLayerConfigPanel } from './index';
 import { DASHBOARDS_MAPS_LAYER_TYPE } from '../../../common';
 import { DocumentLayerConfigPanel } from './documents_config/document_layer_config_panel';
+import { layersTypeIconMap} from '../../model/layersFunctions';
```

Comment:
Space before closing curly brace?
```suggestion
import { layersTypeIconMap } from '../../model/layersFunctions';
```

Reply:
Ack